src/en/components/footer/design.md

```diff
@@ -6,37 +6,81 @@ tags: ['footerEN', 'design']
 date: 'git Last Modified'
 ---
 
-## Footer anatomy – recommended elements
+## Footer anatomy
+
+Footer elements are required for all pages unless noted.
 
 <ol class="anatomy-list">
-  <li>The <strong>GC footer links band</strong> spans the bottom of the page. It contains the official Government of Canada wordmark and links to product terms and conditions and privacy statements, as well as optional links to Canada.ca pages for social media, apps, and about.</li>
-  <li>The <strong>Canada wordmark</strong> is the global identifier of the Government of Canada and one of three official symbols of the Government of Canada's identity.</li>
+  <li>The <strong>contextual band</strong> contains three links specific to your site or product. You can choose which links go there. <em>Optional for all pages.</em></li>
+  <li>The <strong>main band</strong> contains links to common Government of Canada themes. These links cannot be changed. <em>Optional for pages that aren’t standard pages.</em></li>
+  <li>
+    The <strong>GC footer links band</strong> contains the official Government of Canada wordmark and up to five links. Some of the links are optional and some can be modified.
+    <p class="mb-0"><em>Optional for pages that aren’t standard pages:</em></p>
```

Comment:
This is rendering without the spaces, could we fix that?
<img width="770" alt="Screenshot 2025-04-16 at 3 27 34 PM" src="https://github.com/user-attachments/assets/250c4309-2088-4366-952f-957ab1a47f68" />